packages/slice-machine/pages/_app.tsx

```diff
@@ -76,27 +60,16 @@ function MyApp({
     fetcher
   );
 
-  const [sliceMap, setSliceMap] = useState<any | null>(null);
```

Comment:
I asked Hugo about deleting that.
He explained me where it is used and why.

Basically each slice has a personal useState, so React which does not allow conditional hooks needs this.

I think this needs more thinking than just removing this code.

Reply:
I can check with Hugo but, this state isn't shared anywhere, so I don't think this is used anymore.

The store for every slice is part of the librairiesContext construction and that not use this sliceMap.

Reply:
@hypervillain Your help here :)

Reply:
Debt*